docs/develop/pivottables.md

```diff
@@ -0,0 +1,152 @@
+---
+title: 'Work with PivotTables in Office Scripts'
+description: 'The object model for PivotTables in the Office Scripts JavaScript API'
+ms.date: 04/20/2022
+ms.localizationpriority: medium
+---
+
+# Work with PivotTables in Office Scripts
+
+PivotTables let you analyze large collections of data faster. With their power comes complexity. The Office Scripts APIs let you customize a PivotTable to suit your needs, but the scope of the API set makes getting started a challenge. This article demonstrates how to perform common PivotTable tasks and explains important classes and methods.
```

Comment:
```suggestion
PivotTables let you analyze large collections of data faster . With their power comes complexity. The Office Scripts APIs let you customize a PivotTable to suit your needs, but the scope of the API set makes getting started a challenge. This article demonstrates how to perform common PivotTable tasks and explains important classes and methods.
```
Consider changing "faster" to "quickly" or "efficiently" or another appropriate adverb. Comparative adjectives are used to compare differences between the attributes in two nouns. I